src/test/java/com/ozonehis/data/pipelines/BaseJobTest.java

```diff
@@ -103,7 +103,7 @@ public static void beforeAllSuper() {
         try {
             testDir = Files.createTempDirectory(TEST_DIR).toFile().getAbsolutePath();
             exportDir = testDir + "/" + EXPORT_DIR_NAME;
-            TestUtils.pullDockerImage("mekomsolutions/superset-sso", testDir);
```

Comment:
Why not remove the line if it is no longer needed?

Reply:
Left it for future reference